docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+services:
+  postgres:
+    image: postgres:13
+    environment:
+        POSTGRES_PASSWORD: postgres
+        POSTGRES_USER: postgres
+        POSTGRES_DB: dev_db
+    restart: always
+    volumes:
+        - ./pgdata:/var/lib/postgresql/data
```

Comment:
Volume should not use project folder for things like pgdata. It should be defined in `volumes` section of docker compose file like
```
volumes:
  pg_data:
```
and used in service volume section like 
```
  volumes:
    - pg_data:/var/lib/postgresql/data
```